.python-version

```diff
@@ -0,0 +1 @@
+3.13
```

Comment:
I want to double check with you if we need this file in the repo? Do we want to pin one dev version of python? Or might it be easier to drop this